nixos/modules/programs/localsend.nix

```diff
@@ -6,20 +6,32 @@
 }:
 let
   cfg = config.programs.localsend;
-  firewallPort = 53317;
 in
 {
   options.programs.localsend = {
     enable = lib.mkEnableOption "localsend, an open source cross-platform alternative to AirDrop";
 
-    openFirewall = lib.mkEnableOption "opening the firewall port ${toString firewallPort} for receiving files" // {
+    openFirewall = lib.mkEnableOption "opening the firewall port for receiving files" // {
       default = true;
     };
+
+    package = lib.mkPackageOption pkgs "localsend" { };
+
+    # Note that as of version v1.15 (October 2024), the port can not be
+    # changed. However, this may change in the future.
+    port = lib.mkOption {
+      type = lib.types.port;
+      default = 53317;
+      description = "TCP and UDP port used in receive mode for incoming traffic";
+    };
```

Comment:
I can't say that I see the point of the option until support for this is added upstream...